test_email_validation.py

```diff
@@ -0,0 +1,8 @@
+import email_validation
+import unittest
```

Comment:
You may fulfill my previous requirement enabling one of `reorder-python-imports` or `python-import-sorter` in pre-commit's config.